src/ios/SpotifyOAuthPlugin.swift

```diff
@@ -54,6 +58,14 @@ extension URL {
         self.viewController.present(svc, animated: true)
     }
     
+    func scopesToString(scopes: [String]) -> String {
+        var result = ""
+        for scope in scopes {
+            result += scope + " "
+        }
```

Comment:
Instead of the for loop I would use the .joined(separator:) method here.

See https://stackoverflow.com/a/24826511

Reply:
Or rather, when using the idiomatic way of constructing the URL, this method would probably not be required anymore. :)

Reply:
I replaced the method with joined as suggested. I'm not sure about your second point so I'll leave this conversation open in case you have further feedback.